### ISSUE TITLE
Mobile Push Pt1: post-verify page

### ISSUE THREAD

**Issue by davismtl** [mozilla/fxa-features#51]
- Detect UA of where email confirmation was opened to personalize CTA
- Drive to mobile download

Feature Doc: https://docs.google.com/document/d/13AP3rPE4RMKjECBxJzT-OuVyU3DUC1-ggpqUttAJYM4/edit# (in progress)

**Comment by davismtl** [comment]
Work on this began in Toronto. We need to make a feature document.

**Comment by davismtl** [comment]
from mtg:
- We will work on these screens for phase 1 (except sms one)
- Missing is the screens when user signed up on mobile
  https://mozilla.invisionapp.com/share/ZX8RK05G7#/screens/191829446_Verified

**Comment by davismtl** [comment]
@shane-tomlinson @vladikoff @vbudhram @rfk @ryanfeeley 
Please leave feedback on draft of feature document. (link in 1st comment)

**Comment by rfk** [comment]
@shane-tomlinson what's the status on this feature document, can we move this to "building"?

**Comment by shane-tomlinson** [comment]
> @shane-tomlinson what's the status on this feature document, can we move this to "building"?

Yes

**Comment by davismtl** [comment]
from mtg:
- will be merged into next train (77)

**Comment by davismtl** [comment]
from mtg:
- @davismtl needs to look smart banner stats in iTunes Connect
- @ryanfeeley to look at app download button clicks

**Comment by shane-tomlinson** [comment]
And @shane-tomlinson to add some graphs to DataDog for App store clicks between treatment and control group, as well as % of users that see the "sign in" button that actually do so.

**Comment by davismtl** [comment]
from mtg:

- looks really good
- upping to 50% in train 79
- @ryanfeeley has showed that numbers look great.
- @ryanfeeley doing new mockups for more contextual screen between signin vs signup

**Comment by shane-tomlinson** [comment]
> upping to 50% in train 79

Upping the total experiment to 50%, split into two buckets, 25% per bucket.

**Comment by rfk** [comment]
from mtg: latest update not shipped yet, we'll need to revisit the metrics when it does

**Comment by davismtl** [comment]
from mtg: 
- shipped 79 but had to roll it back
- will try to ship it again

**Comment by rfk** [comment]
> will try to ship it again

It's shipped now

**Comment by rfk** [comment]
Dashboard: https://sql.telemetry.mozilla.org/queries/2955/source#5584

**Comment by davismtl** [comment]
from mtg:
- 25% of registration users are now seeing it
- we see a rough 12-15% increase in mobile adoption
- however, we identified an ios tracking bug yesterday which is problably preventing us from properly measuring ios adoption. Chances our though that it even impacted the our metrics before and after this feature roll-out so the improvement rate should still stick.
- @shane-tomlinson we believe that this feature should be rolled out at 100%.
- @davismtl to log results and conclusion in feature document

**Comment by davismtl** [comment]
from mtg: 
- rolling to 100% in train 81
- still behind A/B switch so we can measure in datadog
- shane wants to look into rolling out without the a/b test mechanism without loosing continuity in metrics
- will leave this feature open until we remove the a/b mechanism
- @adavis wil coordinate with @ryanfeeley to log results in Feature Document and write conclusion.

**Comment by davismtl** [comment]
from mtg:
- actually only rolled out to 100% in train 82
- will wait to measure real overall impact
- % of new users connecting a second device has gone up over quarter but difficultly attributable.

**Comment by davismtl** [comment]
from mtg:
- Feeley has questions about whether or not we efficiently rolled out at 100%. The change in metrics was pretty minimal between 5% and 100%.
- @ryanfeeley @shane-tomlinson to dig into the metrics so that we can more easily discuss them during Q1 retrospective on Monday.

**Comment by shane-tomlinson** [comment]
If [this graph](https://app.datadoghq.com/dash/234347/fxa-content---connect-another-device?live=true&page=0&is_auto=false&from_ts=1489742944145&to_ts=1490347744145&tile_size=m&fullscreen=177310190) is zoomed out far enough so that each bar is one day, [like this](https://app.datadoghq.com/dash/234347/fxa-content---connect-another-device?live=true&page=0&is_auto=false&from_ts=1489742944145&to_ts=1490347744145&tile_size=m&fullscreen=177310190), then a different story appears. We released the feature to 25% of the population in trains < 82. Starting with train-82, we released to 100% of people, we would expect to see a 4x increase in the # of users reporting going through the flow. And.... We do:

<img width="218" alt="screen shot 2017-03-23 at 20 03 18" src="https://cloud.githubusercontent.com/assets/848085/24288323/d344ec6a-1074-11e7-9363-a00fa6990448.png">

`\*/`

**Comment by davismtl** [comment]
Following the rollout at 100%, we've observed a huge impact on multi-device adoption. See below

![capture2](https://cloud.githubusercontent.com/assets/19537086/24522266/d9b59abc-1543-11e7-9673-125c0b948300.PNG)

Next steps:
- Need to log results in feature document
- Graduate code from behind feature flags
- Roll-out phase 2 ;)

**Comment by davismtl** [comment]
from mtg: 
- @davismtl added charts and results but needs to add full conclusion.

**Comment by davismtl** [comment]
Added conclusion and charts to feature doc:
https://docs.google.com/document/d/13AP3rPE4RMKjECBxJzT-OuVyU3DUC1-ggpqUttAJYM4/edit#

**Comment by rfk** [comment]
Last steps before we can close this out:
* The final cleanup bugs in https://waffle.io/mozilla/fxa?milestone=FxA-51:%20Mobile%20Push%20Pt1:%20post-verify%20page
* Check that we're removed the feature-flag

**Comment by davismtl** [comment]
from mtg:
- @davismtl to look through remaining cards and decided what must be completed to close this card

**Comment by davismtl** [comment]
CLOSED. FEATURE CARD MOVED TO: 
https://projects.growthhackers.com/s2/26j6dIlN5Wl0YZ3PR0ACZw

**Comment by davismtl** [comment]
done
